app/jobs/document_proofing_job.rb

```diff
@@ -0,0 +1,36 @@
+class DocumentProofingJob < ApplicationJob
+  queue_as :default
+
+  def perform(args)
+    result_id = args[:result_id]
+    encrypted_arguments_ciphertext = args[:encrypted_arguments]
```

Comment:
Does ActiveJob work with keyword arguments?

Reply:
Good call, added in 8ea458094